<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Zomboid Version Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="settingsGroup">
      <property name="title">
       <string>1. Settings</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Manager Storage Folder:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="managerPathEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="browseManagerPathBtn">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Steam 'steamapps' Folder:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="steamappsPathEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="browseSteamappsPathBtn">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Zomboid User Data Folder:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="zomboidUserPathEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="browseZomboidUserPathBtn">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="saveSettingsBtn">
         <property name="text">
          <string>Save Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="manageGroup">
      <property name="title">
       <string>2. Manage Versions</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QListWidget" name="versionListWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="activeVersionLabel">
           <property name="text">
            <string>Detected Version: Unknown</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <!--NEW PLAY BUTTON ADDED -->
         <item>
          <widget class="QPushButton" name="playBtn">
           <property name="text">
            <string>Play</string>
           </property>
          </widget>
         </item>
         <!--NEW PLAY BUTTON -->
         <item>
          <widget class="QPushButton" name="switchToVersionBtn">
           <property name="text">
            <string>Switch to Selected Version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="captureVersionBtn">
           <property name="text">
            <string>Store Current Version</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="instructionsGroup">
      <property name="title">
       <string>3. Instructions to Add a New Version</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="prepareNewVersionBtn">
         <property name="text">
          <string>Step 1: Prepare for New Version (Unlink All)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="instructionsText">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;1. Click the 'Prepare for New Version' button above.&lt;/span&gt; This protects your stored versions.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;2. Open Steam:&lt;/span&gt; Right-click Project Zomboid, go to Properties &amp;gt; Betas.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;3. Change Beta:&lt;/span&gt; Select the desired version and let Steam download the update.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;4. CRITICAL - First Launch:&lt;/span&gt; After downloading, LAUNCH THE GAME ONCE to the main menu, then quit.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;5. Store Version:&lt;/span&gt; Come back here and click the &amp;quot;Store Current Version&amp;quot; button.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#008000;&quot;&gt;6. Activate Version:&lt;/span&gt; The new version is now stored. Select it in the list above and click &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;Switch to Selected Version&amp;quot;&lt;/span&gt; to make it active.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 <resources/>
 <connections/>
</ui>